src/components/output.jsx

```diff
@@ -151,11 +160,26 @@ export default class CellOutput extends React.Component {
     const value = this.props.valueToRender
 
     for (const handler of handlers) {
-      if (handler.shouldHandle(value)) {
-        const resultElem = handler.render(value)
-        if (resultElem !== undefined) {
+      let shouldHandle = false
+      try {
+        shouldHandle = handler.shouldHandle(value)
+      } catch (error) {
+        console.log('An output handler threw an exception in shouldHandle')
```

Comment:
We should probably propagate this error up somehow... ditto for the error below.

Reply:
I thought about that, but it's sort of a "soft error".  If one of the output handlers borks, it should gracefully fall back to the generic handlers that should hopefully do a decent job of displaying just about anything.  I think that's better than an exception.  I guess the best thing would be to get the generic output *plus* a little inline message above it saying that an output handler failed.  That sort of goes back to the broader question of how to display errors (#166, #38)